sdks/java/core/src/main/java/org/apache/beam/sdk/values/Row.java

```diff
@@ -836,10 +838,10 @@ public int nextFieldId() {
     }
 
     @Internal
-    public Row withFieldValueGetters(
-        Factory<List<FieldValueGetter>> fieldValueGetterFactory, Object getterTarget) {
+    public <T extends @NonNull Object> Row withFieldValueGetters(
```

Comment:
unclear on this bound as well

Reply:
I could change it to `<@NonNull T>` I suppose - it compiles, but then my IDE complains that "Non-null type argument is expected" on the `T` parametor of the `FieldValueGetter<T, ?>` type

Reply:
I think I prefer @NonNull T (is this simply a weakness in the IDE?), but I guess I don't have strong feelings here.